intermediate_data_model.py

```diff
@@ -0,0 +1,436 @@
+import re
+
+import utils
+from purchase_invoice import SupplierItem, PurchaseInvoice
+from settings import NKK_ACCOUNTS, KORNKRAFT_ACCOUNTS
+
+
+class InvoiceIntermediateModel:
+    def __init__(self, purchase_invoice, generation_type, lines):
```

Comment:
`generation_type` should be named `supplier` .

Reply:
Fixed